website/docs/basics/block-structure.md

```diff
@@ -36,6 +39,12 @@ In this folder, you'll define all the custom JavaScript functionality for your b
 ### components
 Components folder holds three files `heading-options.js`, `heading-editor.js`, and `heading-toolbars.js`. Each of these files represents a part of the Block Editor block that is used in the editor. We have separated options, editor, and toolbar into separate components for readability and reusing components in different projects.
 
+### docs
+Docs folder contains everything that a block needs to be run inside the Storybook. This part is optional. But it will speed up your development time and provide you with the much needed documentation.
```

Comment:
```suggestion
Contains everything that a block needs to be run inside the Storybook. This part is optional. However, it will speed up your development time and provide you with the much needed documentation.
```